installer/src/components/AccountForm/AccountForm.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { getBackendUri } from '../../constants/backendUri';
+import { hlsPluginName } from '../../constants/constants';
+import { Plugin } from '../../constants/interface';
+import LoadingSpinner from '../LoadingSpinner/LoadingSpinner';
+import NoPluginMessage from '../NoPluginMessage/NoPluginMessage';
+
+const AccountForm = () => {
+  // Call useEffect to call function to get account Name
+  const [accountName, setAccountName] = useState<string>("");
+  const [accountSid, setAccountSid] = useState<string>("");
+  const [accountAuthToken, setAccountAuthToken] = useState<string>("");
+  const [pluginFriendlyName, setPluginFriendlyName] = useState<string>("");
+  const [pluginSid, setPluginSid] = useState<string>("");
+  const [hlsPluginExists, setHlsPluginExists] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const handleDeployment = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+    if (accountSid && accountAuthToken) {
+      // This call creates the workspace and routers via APIs
```

Comment:
you wouldn't need these comments if you move these fetches into named functions and descriptively name them with what the comments say :) comments should not about be written for the "what", they should be written for the "why"